pom.xml

```diff
@@ -128,6 +128,12 @@
         <artifactId>tomcat-jaspic-api</artifactId>
         <version>${tomcat.version}</version>
       </dependency>
+      <dependency>
```

Comment:
do we need it? isnt the jaxrs client enough?

Reply:
will check the dependencies and minimize if possible.

Reply:
ok, removed the dep.